src/pages/yeonokkang/Login/LoginValidation.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import LoginBtn from './LoginBtn';
+import Input from './Input';
+
+class LoginInput extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: '',
+      pw: '',
+      isActivate: false,
+    };
+  }
+
+  handleIdInput = event => {
+    this.setState({
+      id: event.target.value,
+    });
+  };
+
+  handlePwInput = event => {
+    this.setState({
+      pw: event.target.value,
+    });
+  };
```

Comment:
```suggestion
 handleInputs = e => {
    this.setState({
      [e.target.name]: e.target.value,
    });
  };
```
- 이런 식으로 함수를 간단하게 하나로 줄일 수 있습니다!
- 단, 각 input 태그에 해당하는 name 속성을 부여해 주셔야 합니다!

Reply:
오 더 간단하고 좋은 방법이네요!! Input을 하나로 컴포넌트화했기 때문에 text값을 기준으로 삼항연산자로 코드 변경해봤습니다 :) 덕분에 코드가 훨씬 깔끔하고 좋아졌어요! 감사합니당🤓